sandboxed_api/bazel/filewrapper.cc

```diff
@@ -79,6 +79,9 @@ void FWriteCEscapedC(int c, FILE* out) {
       case '\\':
         fputc('\\', out);
         break;
+      case '?':
+        fputc('?', out);
```

Comment:
This could be changed to:
```c
      case '\"':
      case '\'':
      case '\\':
      case '?':
        fputc(c, out);
        break;
```

Reply:
I'm fine with either. Need to be careful not to apply this optimization to the first few cases, though :)

Reply:
I like the suggestion and I have adjusted the PR.